### ISSUE TITLE
ADA - Buttons need text descriptions added

### ISSUE THREAD

**Issue by katelyndavis** [immregistries/mqe#78]
[Overview]
Buttons must have discernible text that clearly describes the destination, purpose, function, or action for screen reader users.

Buttons that need this fix include

-  the date forward/backward buttons on dashboard screen - 3 or 4 of these
![image](https://user-images.githubusercontent.com/37371748/42707580-a9bd854a-86a8-11e8-934e-d7f9093c86f8.png)

- trash button on upload file screen

[Requirements]
[https://dequeuniversity.com/rules/axe/3.0/button-name?application=AxeChrome](url)

[Dev Notes]
To solve this violation, you need to:
Fix at least one (1) of these issues:
Element has a value attribute and the value attribute is empty
Element has no value attribute or the value attribute is empty
Element does not have inner text that is visible to screen readers
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty or not visible
Element's default semantics were not overridden with role="presentation"
Element's default semantics were not overridden with role="none"
Element has no title attribute or the title attribute is empty

[Testing]
